charts/consul/test/unit/connect-inject-deployment.bats

```diff
@@ -708,20 +697,20 @@ load _helpers
 
   local actual=$(echo $object |
     yq 'any(contains("enable-k8s-namespace-mirroring"))' | tee /dev/stderr)
-  [ "${actual}" = "false" ]
+  [ "${actual}" = "true" ]
 
   local actual=$(echo $object |
     yq 'any(contains("k8s-namespace-mirroring-prefix"))' | tee /dev/stderr)
   [ "${actual}" = "false" ]
 }
 
-@test "connectInject/Deployment: mirroring options set with .connectInject.consulNamespaces.mirroringK8S=true" {
+@test "connectInject/Deployment: mirroring options omitted with .connectInject.consulNamespaces.mirroringK8S=false" {
```

Comment:
The condition this checked is now covered by the test on line 681. I changed it to check the opposite case as it is no longer the default behavior.